tests/chainer_tests/test_cuda.py

```diff
@@ -8,6 +8,18 @@
 from chainer.testing import attr
 
 
+class TestDummyDeviceType(unittest.TestCase):
+
+    def test_int(self):
+        self.assertEqual(cuda.DummyDeviceType(), -1)
```

Comment:
This test [fails](https://travis-ci.org/pfnet/chainer/jobs/121204893). Use `int(cuda.DummyDeviceType())`